Documentation/gettingstarted/istio.rst

```diff
@@ -455,56 +464,56 @@ whitelisted:
 
 .. literalinclude:: ../../examples/kubernetes-istio/bookinfo-productpage-v2-policy.yaml
 
-Create the ``productpage v2`` service and its updated
-CiliumNetworkPolicy and delete ``productpage v1``:
-
-.. parsed-literal::
+Because ``productpage v2`` sends messages into Kafka, we must first
+deploy a Kafka broker:
 
-    $ curl -s \ |SCM_WEB|\/examples/kubernetes-istio/bookinfo-productpage-v2.yaml | \\
-          istioctl kube-inject --injectConfigMapName istio-inject -f - | \\
-          sed -f ./cilium-kube-inject.sed | \\
+.. TODO: Re-enable sidecar injection after we support Kafka with mTLS.
+    $ curl -s \ |SCM_WEB|\/examples/kubernetes-istio/kafka-v1.yaml | \\
+          istioctl kube-inject --injectConfigFile istio-inject-config.yaml -f - | \\
           kubectl create -f -
-    ciliumnetworkpolicy "productpage-v2" created
-    deployment "productpage-v2" created
 
-    $ kubectl delete -f \ |SCM_WEB|\/examples/kubernetes-istio/bookinfo-productpage-v1.yaml
+.. parsed-literal::
 
-Check the progress of the deployment (every service should have an
-``AVAILABLE`` count of ``1``):
+    $ kubectl create -f \ |SCM_WEB|\/examples/kubernetes-istio/kafka-v1.yaml
+    service "kafka" created
+    ciliumnetworkpolicy "kafka-authaudit" created
+    statefulset "kafka-v1" created
+
+Wait until the ``kafka-v1-0`` pod is ready, i.e. until it has a
+``READY`` count of ``2/2``:
 
 ::
 
-    $ kubectl get deployments -n default
-    NAME             DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-    details-v1       1         1         1            1           15m
-    productpage-v2   1         1         1            1           1m
-    ratings-v1       1         1         1            1           10m
-    reviews-v1       1         1         1            1           15m
-    reviews-v2       1         1         1            1           10m
+    $ kubectl get pods -n default -l app=kafka
+    NAME         READY     STATUS    RESTARTS   AGE
+    kafka-v1-0   2/2       Running   0          21m
```

Comment:
I got `1/1`, not `2/2`
```
NAME         READY     STATUS    RESTARTS   AGE
kafka-v1-0   1/1       Running   0          2m
```

Reply:
Fixed. It should be 1/1 now that we don't inject a sidecar into the Kafka pod anymore.